Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml

```diff
@@ -177,6 +177,9 @@
             </ns4:typeSpecifier>
         </ns4:element>
     </ns4:typeInfo>
+    <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.ResultComponent" retrievable="false" baseType="QDM.Component">
```

Comment:
Looks good, although I wonder if `ResultComponent` is the best name. Aren't most uses of component like a result even out side of lab tests? I do not have a good suggestion for another name. If you're comfortable with this name, then this PR looks good to me.